packages/sirius-web/frontend/sirius-web-application/src/modals/new-document/NewDocumentModal.types.ts

```diff
@@ -59,6 +59,11 @@ export interface GQLCreateDocumentPayload {
 
 export interface GQLSuccessPayload extends GQLCreateDocumentPayload {
   id: string;
+  document: GQLDocument;
```

Comment:
The payload is this case should not be a GQLSuccessPayload but a GQLCreateDocumentSuccessPayload